generation/internal/node.go

```diff
@@ -22,6 +22,13 @@ func nodePackageManager(ls labels.LabelSet) string {
 
 func nodeRunCommand(ls labels.LabelSet, task string) string {
 	if task == "test" {
+		// don't fail for default test value for new node projects
+		for _, label := range ls {
```

Comment:
Rather than check all labels, it looks like we should only need to check the [`DepsNode`](https://github.com/CircleCI-Public/circleci-config/blob/main/labeling/internal/node.go#L24) label, since that's where the package.json data gets parsed?